Please check exports and imports fields in package.json, because they fail to build.

Module not found: Error: Package path ./dist/loader/css/react-spinner-loader.css is not exported from package /home/runner/work/goit-react-hw-04-hooks-images/goit-react-hw-04-hooks-images/node_modules/react-loader-spinner (see exports field in /home/runner/work/goit-react-hw-04-hooks-images/goit-react-hw-04-hooks-images/node_modules/react-loader-spinner/package.json)

--- a/package.json
+++ b/package.json
@@ -59,6 +59,7 @@
     "@typescript-eslint/eslint-plugin": "^5.8.0",
     "@typescript-eslint/parser": "^5.8.0",
     "copyfiles": "^2.4.1",
+    "docz": "^2.3.1",
     "eslint": "^7.32.0",
     "eslint-config-standard": "^16.0.3",
     "eslint-plugin-import": "^2.25.3",

--- a/package.json
+++ b/package.json
@@ -7,8 +7,14 @@
   "main": "dist/index.js",
   "module": "dist/esm/index.js",
   "exports": {
-    "require": "./dist/index.js",
-    "import": "./dist/esm/index.js"
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./dist/esm/index.js"
+    },
+    "./dist/loader/css/react-spinner-loader.css":  "./dist/loader/css/react-spinner-loader.css",
+    "./dist/loader/css/CradleLoader.css":  "./dist/loader/css/CradleLoader.css",
+    "./dist/loader/css/Plane.css":  "./dist/loader/css/Plane.css",
+    "./dist/loader/css/Triangle.css":  "./dist/loader/css/Triangle.css"
   },
   "types": "dist/index.d.ts",
   "author": {
@@ -53,7 +59,6 @@
     "@typescript-eslint/eslint-plugin": "^5.8.0",
     "@typescript-eslint/parser": "^5.8.0",
     "copyfiles": "^2.4.1",
-    "docz": "^2.3.1",
     "eslint": "^7.32.0",
     "eslint-config-standard": "^16.0.3",
     "eslint-plugin-import": "^2.25.3",

